node-labeler/namespace.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
should the namespace go into `cluster-scope/base/core/namespaces/` ?

Reply:
Yes it should. If you fix that here, make sure to update the `ocp-prod` overlay as well to include the namespace.